### PR TITLE
Add tooltip to the enrollments.

### PR DESCRIPTION
**Pull request by m6jones**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ addons:
   chrome: stable
 before_script:
 - npm run lint
-- npm run gulp
```
Comment (m6jones) [inline .travis.yml]: running this failed the tests. Don't need to run it in travis. More manual updates to language files.

File: bower.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "d2l-organization-hm-behavior": "Brightspace/organization-hm-behavior#^2.0.1",
     "d2l-organizations": "BrightspaceHypermediaComponents/organizations#^0.1.8-hybrid",
     "d2l-status-indicator": "^2.0.0",
+    "d2l-tooltip": "^2.0.4",
```
Comment (dbatiste) [inline bower.json]: Note: the version that's needed is `2.0.5` (which should be the version that's picked up as per semver).  I noticed @jstefaniuk-d2l updated tooltips about an hour after this as well, I _think_ for something unrelated.

**Comment by dbatiste** [comment]
Looks like we're still using `fit-content` for the `width`, which I think won't work in Edge and IE11.  I think we should look into/address that if possible.

**Comment by dbatiste** [comment]
Nit: ideally we should update the version of card to the version that's needed to properly support tooltips in the footer.  (`4.1.17`)

**Pull request merged**